build.nix

```diff
@@ -3,7 +3,10 @@ src:
   cargoBuild
 , #| What command to run during the test phase
   cargoTest ? "cargo test --$CARGO_BUILD_PROFILE"
+, #| What command to run during the optional doc phase
+  cargoDoc ? "cargo doc --offline --frozen"
```

Comment:
`--frozen` _may_ fail in some circumstances when cargo needs to update the Cargo.lock for weird reasons, even though it won't access the network